sources/javascripts/concat/global/main.js

```diff
@@ -118,6 +118,35 @@
     $('.content-formatted table').wrap('<div class="table-container"></div>');
   };
 
+  // Change product image position on narrower screens (mobile devices)
+  var handleProductPageContent = function () {
+    $(document).ready(function () {
+      changeProductImagePos();
+    });
+
+    $(window).resize(debounce(function () {
+      changeProductImagePos();
+    }, 25));
+
+    var changeProductImagePos = function () {
+      var productGallery = $('.js-product-gallery');
+      var productImageContentBox = $('.js-content-item-box');
+      var buyBtnContent = $('.js-buy-btn-content');
```

Comment:
Mayby its more readable to add class like `js-product-content-right` to product right column wrapper and use it here (like in Santiago https://github.com/Voog/design-santiago/pull/230/files#diff-96d8063d0f25fce87494d9213aa996cf6bf3e4288934f4e77634d45ee6abfe82R1075)